protoarray/src/main/java/tech/pegasys/artemis/protoarray/ProtoArrayForkChoiceStrategy.java

```diff
@@ -271,26 +272,20 @@ public boolean containsBlock(Bytes32 blockRoot) {
    * </ul>
    *
    * @param indices
-   * @param votes
+   * @param store
    * @param oldBalances
    * @param newBalances
    * @return
    */
   static List<Long> computeDeltas(
+      MutableStore store,
       Map<Bytes32, Integer> indices,
-      ElasticList<VoteTracker> votes,
       List<UnsignedLong> oldBalances,
       List<UnsignedLong> newBalances) {
     List<Long> deltas = new ArrayList<>(Collections.nCopies(indices.size(), 0L));
 
-    for (int validatorIndex = 0; validatorIndex < votes.size(); validatorIndex++) {
-      VoteTracker vote = votes.get(validatorIndex);
-
```

Comment:
Instead of continuing whenever the validator has never voted, we simply don't iterate through those validators now, which is also more efficient.